pages.fr/common/ffuf.md

```diff
@@ -0,0 +1,25 @@
+# ffuf
+
+> Un outil d'énumération web rapide écrit en Go.
```

Comment:
I think that nobody will understand `outil d'énumération web`. 😄 
I suggest (I have remove "simple" to simplify the sentence... and because it's subjective, isn't it? 😇): 
```suggestion
> Un outil de tests aléatoires web de type _fuzzer_ écrit en Go.
```
Cf. https://fr.wikipedia.org/wiki/Fuzzing

Reply:
(by the way, "écrit en Go" is not very useful as an end-user, but I assume that we don't want to change all other page translations in this pull request) 😇

Reply:
Yeah I kept `rapide` only because it was in the English page but of course it's subjective and promotional.

But I fee like `Un outil d'énumération web` is way more understandable than `Un outil de tests aléatoires web`, that's even why you were forced to add `de type _fuzzer_`.

And using _fuzz_ is a bit abusive, it's not any fuzzing like AFL, it's not injecting any random stuff etc. it's only replacing a keyword by words in a wordlist to do enumeration, trying to enumerate existing pages, repositories, extensions, sub-domains, etc. So even in the English page it would vote for removing `fuzzer` and replace it by `web enumeration tool`.